tools/src/main/java/org/apache/kafka/tools/streams/StreamsGroupCommand.java

```diff
@@ -830,6 +831,21 @@ Map<String, List<String>> retrieveInternalTopics(List<String> groupIds) {
                 }
             } catch (InterruptedException | ExecutionException e) {
                 if (e.getCause() instanceof UnsupportedVersionException) {
+                    List<String> internalTopics = new ArrayList<>();
+                    try {
+                        Set<String> allTopics = adminClient.listTopics().names().get();
+                        for (String topic : allTopics) {
+                            for (String groupId : groupIds) {
+                                if (isInferredInternalTopic(topic, groupId)) {
+                                    internalTopics.add(topic);
+                                    break;
+                                }
+                            }
+                        }
+                        System.out.println("Internal Topics: (" + String.join(",", internalTopics) + ").");
```

Comment:
We could use the stream api to align the style with this whole class.
For example:
```suggestion
                    String internalTopics;
                    try {
                        Set<String> allTopics = adminClient.listTopics().names().get();
                        internalTopics = allTopics.stream()
                            .filter(topic -> groupIds.stream().anyMatch(groupId -> isInferredInternalTopic(topic, groupId)))
                            .collect(Collectors.joining(","));
                        System.out.println("Internal Topics: (" + internalTopics + ").");
```

Reply:
Thanks for the comment! Just revised the method:
1. use Stream API
2. Combine print internal topics in the `printError` message.